internal/storage/blob/store.go

```diff
@@ -228,6 +231,11 @@ func (s *Store) updateIndex(ctx context.Context) error {
 	var p *policyv1.Policy
 	var event storage.Event
 	for _, f := range changes.updateOrAdd {
+		if strings.HasSuffix(f, filepath.Join(schema.Directory, schema.File)) {
```

Comment:
Is there a way we can ensure that this matches only  `<ROOT>/_schemas/schema.yaml` and nothing else like  `<ROOT>/some/path/_schemas/schema.yaml` etc?